apis/core/lib/handlers/dataset.ts

```diff
@@ -26,26 +32,44 @@ export const put = protectedResource(
   }),
 )
 
-export const loadCubes: Enrichment = async (req, dataset) => {
-  const shapeQuads = await loadCubeShapes(dataset, streamClient)
-  let graph = ''
+export const get = protectedResource(asyncMiddleware(async (req, res) => {
+  const dataset = await req.hydra.resource.clownface()
+  const shapeStream = await loadCubeShapes(req.hydra.resource.term, streamClient)
+  const outStream = new PassThrough({
+    objectMode: true,
+  })
+
+  const types = clownface({
+    dataset: req.hydra.api.dataset,
+    term: dataset.out(rdf.type).terms,
+  })
+  const linkedResources = await loadLinkedResources(dataset, types.out(query.include).toArray(), req.app.sparql)
```

Comment:
This could be improved by making `loadLinkedResources` a stream and having it called automatically behind the scene. That way `req.hydra.resource.clownface()` would also be replaced with `req.hydra.resource.quadStream()` to avoid unnecessarily materialising all quads